packages/data-stores/src/reader/mutations/use-site-unsubscribe-mutation.ts

```diff
@@ -14,11 +18,16 @@ type UnsubscribeResponse = {
 	subscription?: null;
 };
 
-const useSiteUnsubscribeMutation = () => {
+const useSiteUnsubscribeMutation = ( blog_id?: string ) => {
 	const { isLoggedIn } = useIsLoggedIn();
 	const queryClient = useQueryClient();
 	const siteSubscriptionsCacheKey = useCacheKey( [ 'read', 'site-subscriptions' ] );
 	const subscriptionsCountCacheKey = useCacheKey( [ 'read', 'subscriptions-count' ] );
+	const siteSubscriptionDetailsCacheKey = useCacheKey( [
+		'read',
+		'site-subscription-details',
+		...( blog_id ? [ blog_id ] : [] ),
```

Comment:
I think in terms of the cache key, it is perfectly fine to just spit out `blog_id` as is, even if it is `undefined`. But I suppose there is nothing wrong with this solution too. 🙂

Reply:
Thanks for the suggestion, Maciej!

The reason for the construction above is that I didn't want to have the empty string, nor word "undefined" as a part of the cache key. But I agree that your proposal makes the code cleaner.